dev-lang/rust/files/1.48.0-gentoo-musl-target-specs.patch

```diff
@@ -27,50 +27,50 @@ index 0000000..420fe7c
 --- /dev/null
```

Comment:
Nit: the patch file name should be '1.49.0-gentoo...`.

Reply:
Sure, will rename. Thanks.